pyslackersweb/website/tasks.py

```diff
@@ -72,13 +72,15 @@ class Channel:
 
         repositories.sort(key=lambda r: r.stars, reverse=True)
 
-        await redis.set(cache_key, json.dumps([x.__dict__ for x in repositories[:6]]))
-
+        await redis.set(
+            cache_key, json.dumps([dataclasses.asdict(repo) for repo in repositories[:6]])
```

Comment:
Nice usage, I wasn't aware of `dataclasses.asdict` 👍

Reply:
I wonder if we should model the object as a dataclass and populate it in the view as a typed class instead of a dict? Probably a follow-up item.

Reply:
For storing into redis we don't really have a choice. For the template that sounds like a good idea